deploy/determined_deploy/aws/cli.py

```diff
@@ -152,6 +157,7 @@ def deploy_aws(args: argparse.Namespace) -> None:
 
     det_configs = {
         constants.cloudformation.KEYPAIR: args.keypair,
+        constants.cloudformation.ENABLE_CORS: "true" if args.enable_cors else "false",
```

Comment:
Can you use the `store_true` action for `enable_cors` instead of this check?

Reply:
sg this would just mean more changes in the CI since we no longer can do 

```
 enable-cors: <<parameters.enable-cors>>
```

and would need to write a conditional CI job, duplicate the det-deploy command and pass the `enable_cors` flag in one and omit it in the other or create two separate deploy-aws-cluster jobs and forgo the enable_cors parameter in CI

Reply:
I know it's more of a CI limitation but since we use det-deploy CLI the cli syntax affects our CI. would it be back if we keep it simpler for CI? it'd look like something like https://github.com/determined-ai/determined/pull/1439/commits/511e9c57c66d8c828e9f7fb435db0a98bd2dec05 @rb-determined-ai  any thoughts on the CI portion

Reply:
https://support.circleci.com/hc/en-us/articles/360043638052-Conditional-steps-in-jobs-and-conditional-workflows

Reply:
If it's going to cause major code changes then we don't have to do it. It's not that big of a deal

Reply:
You could support a `--no-enable-cors` flag as well, and instead of `--enable_cors <<parameters.enable-cors>>` you could just have `<<parameters.enable-cors>>`, which is either `--no-enable-cors` or `--enable-cors`.

But I think a good user interface trumps easy CI in this case, and `--enable-cors true` seems awfully strange for a cli.

Reply:
I think this is the change we want then https://github.com/determined-ai/determined/pull/1439/commits/511e9c57c66d8c828e9f7fb435db0a98bd2dec05 I'